scriptmodules/ports/openpht.sh

```diff
@@ -42,6 +43,7 @@ function install_bin_openpht() {
 
 function remove_openpht() {
     aptRemove openpht
+    rp_callModule openpht depends remove
```

Comment:
Please remove this line (we don't force removal of package dependencies and this will remove pulseaudio)